src/rpc/RPCHelpers.cpp

```diff
@@ -1273,6 +1273,23 @@ specifiesCurrentOrClosedLedger(boost::json::object const& request)
     return false;
 }
 
+bool
+isAdminCmd(std::string const& method, boost::json::object const& request)
+{
+    auto const isFieldSet = [&request](auto const field) {
+        return request.contains(field) and request.at(field).is_bool() and request.at(field).as_bool();
+    };
+    if (method == JS(ledger)) {
+        if (isFieldSet(JS(full)) or isFieldSet(JS(accounts)) or isFieldSet(JS(type))) {
+            return true;
+        }
+    }
+
+    if (method == JS(feature) and request.contains(JS(vetoed)))
```

Comment:
Should we allow forwarding if vetoed is false?

Reply:
Absolutely not. Clio should never be connecting to validator nodes in the first place.

Reply:
I see. The `vetoed` is to really vote on something.